### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joelibaceta**
Changes:

Because, its works in Mac & Linux too, we could generalize the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ The current focus of the Bosque project is core language design. As a result the
 
 ### Requirements
 
-#### Windows
-
 In order to build the language for Windows the following are needed:
```
Comment (mrkmarron) [inline README.md]: We should delete "for Windows" here as well.

File: README.md [hunk 2]
```diff
@@ -82,13 +82,13 @@ The current focus of the Bosque project is core language design. As a result the
 
 ### Requirements
 
-#### Windows
-
 In order to build the language for Windows the following are needed:
 
-- The LTS version of [node.js](https://nodejs.org/en/download/) for Windows (x64)
+- 64 bits Operating System
```
Comment (mrkmarron) [inline README.md]: "bit" or "bits" -- I favor "bit" but either is ok with me

File: README.md (hunk 1 above)
Comment (joelibaceta) [inline README.md]:
```suggestion
In order to build the language the following are needed:
```

File: README.md [hunk 3]
```diff
@@ -82,13 +82,13 @@ The current focus of the Bosque project is core language design. As a result the
...
+In order to build the language the following are needed:
 
-In order to build the language for Windows the following are needed:
-
-- The LTS version of [node.js](https://nodejs.org/en/download/) for Windows (x64)
+- 64 bits Operating System
+- The LTS version of [node.js](https://nodejs.org/en/download/) ( According to your OS )
 - Typescript (install with: `npm i typescript -g`)
 
+
```
Comment (joelibaceta) [inline README.md]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (joelibaceta) [reply README.md]:
```suggestion
- 64 bit Operating System
```

Comment (joelibaceta) [reply README.md]: bit is ok for me

File: README.md (hunk 1 above)
Comment (joelibaceta) [reply README.md]: Removed

**Pull request merged**